java8_time/duration.go

```diff
@@ -18,8 +18,8 @@
 package java8_time
 
 type Duration struct {
-	Second int64 `hessian:"second"`
-	Nano   int32 `hessian:"nano"`
+	Seconds int64 `hessian:"seconds"`
```

Comment:
I can not agree with u. Pls do not change the var name.

Reply:
This is NOT rename, but correct the wrong name to the correct one which java used.

Reply:
got it.